src/main/navigation/navigation_fixedwing.c

```diff
@@ -213,6 +213,11 @@ void resetFixedWingPositionController(void)
     pt1FilterReset(&fwPosControllerCorrectionFilterState, 0.0f);
 }
 
+static int8_t loiterDirection(void) {
```

Comment:
I think it is unnecessarily verbose to use a function for this. Could you define instead a variable at the beginning of `calculateVirtualPositionTarget_FW` like:

`int8_t loiterDirectionSign = pidProfile()->loiter_direction == NAV_LOITER_LEFT ? -1 : 1;`

Reply:
I would prefer to leave it in function because in the future I would like to:
>have an option to change the direction ... wit a channel switch

> have an option to change it in flight by using the rudder stick

Besides, I use it also in function updatePositionHeadingController_FW

Reply:
@hali9 That seems reasonable too